.github/workflows/lint.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        emacs_version: [26, 27, 28, "master"]
+        emacs_version: [27, 28, "master"]
```

Comment:
Emacs 26 support removed. Looks like that version is so old the executable-find check cannot check remote tramp paths. Rather than adding a workaround I think its fine to just remove it. Supporting almost 3 past version releases seems excessive IMO :thinking:.

Reply:
I think that is okay. Emacs 27 is available on the latest Ubuntu LTS release and the latest Ubuntu LTS release has been out for more than a year, so it is fine to drop older versions.